simulation/adx_gyro_accelerometer/src/main/native/include/ADXRS450_SpiGyroWrapperData.h

```diff
@@ -32,12 +34,17 @@ class ADXRS450_SpiGyroWrapper {
   void SetAngle(double angle);
 
  private:
-  const int m_port;
+  int m_port;
+  int m_readCallbackId;
+  int m_autoReceiveReadCallbackId;
 
   wpi::mutex m_registerMutex;
   std::atomic<double> m_angle{0.0};
+  std::atomic<double> m_angle_diff{0.0};
```

Comment:
Our style guide requires mixed case, so `m_angleDiff` instead of `m_angle_diff`.